week-2-practise/level-3/6-map.js

```diff
@@ -4,18 +4,19 @@
 //    - every resulting item run through the `captialise` function
 
 function tidyUpString(str) {
-  return str
+    var mentorsTidy = str
     .trim()
     .toLowerCase()
     .replace("/", "");
+    return captialise(mentorsTidy);
```

Comment:
What if we don't want to modify tidyUpString function, as specified in the instructions?